src/compute-types/src/dataflows.rs

```diff
@@ -72,10 +71,10 @@ pub struct DataflowDescription<P, S: 'static = (), T = mz_repr::Timestamp> {
     pub initial_storage_as_of: Option<Antichain<T>>,
     /// The schedule of REFRESH materialized views.
     pub refresh_schedule: Option<RefreshSchedule>,
-    /// Human readable name
+    /// Human-readable name
     pub debug_name: String,
-    /// Information about the dataflow used to determine if dataflow expiration can be enabled.
-    pub dataflow_expiration_desc: DataflowExpirationDesc<T>,
+    /// Description of how the dataflow's progress relates to wall-clock time. None for unknown.
+    pub time_dependence: Option<TimeDependence>,
```

Comment:
Is it possible to unify `None` and `Some(TimeDependence::Indeterminate)`? Naively it seems like they're quite similar?

Reply:
They help separate out the "unset" case from the tried to determine and found Indeterminate case.

Reply:
Unified, and turned `TimeDependence` into a struct. Thanks for the nudge.